latex.tplx

```diff
@@ -1,3 +1,28 @@
+%=====================================
+% Solution about utf8x problem
+% https://github.com/jupyter/nbconvert/issues/530#issuecomment-303034557
+%=====================================
+
+((*- extends 'article.tplx' -*))
```

Comment:
I think this is why the build fails on Travis: A template can extend at most one other template [(Jinja docs)](http://jinja.pocoo.org/docs/2.10/templates/#extends), and we are already extending `ipython_style.tplx` later on. We can probably just move all this after the existing `((* extends cell_style *))` instruction. That template also has a `packages` block, so it should work.